package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "make-manifest",
-  "version": "1.0.4",
+  "version": "1.1.0",
```

Comment:
As we are dropping support from nodejs@10 to nodejs@14 I will suggest to do a Major release, as we can break legacy projects

Reply:
As well it will require to update the engines section in the pkg accordingly, [here](https://github.com/guidesmiths/make-manifest/blob/b31d48cd6bc1d5d642c0b2b49cbf88211a7c63bd/package.json#L7)

Reply:
You're right Ulises. I've removed support for node@10 and node@12 because the GitHub action was not running as it said that it didn't support that engine. I'll fix the engines and do a major version change.